packages/webapp/components/CookieBanner.tsx

```diff
@@ -39,6 +39,7 @@ export default function CookieBanner({
     >
       <ModalCloseButton onClick={close} />
       <CookieIcon
+        size="auto"
         className="hidden laptop:block mb-4 text-theme-label-primary"
         style={{ fontSize: '3.5rem' }}
```

Comment:
One of the reasons we did the refactoring of icons was to be consistent with how we implemented the icons. If we're gonna keep using font-size for changing its actual size, incoming developers would have doubts about which one to use.

We have moved all other implementations to use width and height. So kindly change the style property here to override the width and height style.